packages/tracing-internal/src/browser/browserTracingIntegration.ts

```diff
@@ -43,34 +33,28 @@ export const BROWSER_TRACING_INTEGRATION_ID = 'BrowserTracing';
 /** Options for Browser Tracing integration */
 export interface BrowserTracingOptions {
   /**
-   * The time to wait in ms until the transaction will be finished during an idle state. An idle state is defined
-   * by a moment where there are no in-progress spans.
-   *
-   * The transaction will use the end timestamp of the last finished span as the endtime for the transaction.
-   * If there are still active spans when this the `idleTimeout` is set, the `idleTimeout` will get reset.
-   * Time is in ms.
+   * The time that has to pass without any span being created.
+   * If this time is exceeded, the idle span will finish.
    *
-   * Default: 1000
+   * Default: 1000ms
```

Comment:
```suggestion
   * Default: `1000` (milliseconds)
```

Same for other comments below.